docs/content/latest/yugabyte-cloud/cloud-admin/cloud-billing-costs.md

```diff
@@ -239,3 +239,39 @@ The free allowance for data out transfers is 10GB per month for every 1 vCPU per
   - Re-write existing data to your database deployment.
 
 - If possible, configure your client driver to use wire protocol compression to communicate with the YugabyteDB cluster. Yugabyte Cloud always compresses intra-cluster communication.
+
+## Paused cluster costs
+
+Yugabyte suspends [instance vCPU capacity costs](#instance-vcpu-capacity-costs) for paused clusters. Paused clusters are billed for [disk storage](#disk-storage-cost) and [backup storage](#backup-storage-costs) at the standard rates; this cost includes any storage that is normally covered by your running cluster free allowances.
+
+For example, suppose you have a cluster with following configuration:
+
+- Total number of vCPUs: 1 node * 4 vCPU = 4 vCPUs
+- Disk storage used: 200 GB
+- Backup storage used: 400 GB
+
+While active, disk and backup storage are covered by the free allowance, and the cluster is charged at the following rate:
+
+```output
```

Comment:
Any particular reason this is formatted as console output? Does it get displayed in a console?